github/MainClass.pyi

```diff
@@ -32,10 +30,12 @@ class Github:
         login_or_token: Optional[str] = ...,
         password: Optional[str] = ...,
         jwt: Optional[str] = ...,
+        app_id: Optional[Union[int, str]] = ...,
+        app_private_key: Optional[str] = ...,
+        app_installation_id: Optional[int] = ...,
+        app_token_permissions: Optional[dict] = ...,
```

Comment:
dict containing what types?

Reply:
I'll change those to `Dict[str, str]`. The GitHub reference just states `object`